jdaviz/tests/test_app.py

```diff
@@ -232,7 +232,7 @@ def test_to_unit(cubeviz_helper):
 
     # will be a uniform array since not wavelength dependent
     # so test first value in array
-    assert np.allclose(value[0], 4.800000041882413e-08)
+    assert np.allclose(value[0], 8e-11)

```

Comment:
wait, is this change expected?  Is this because of the change to remove `aperture_area_along_spectral`?  (cc @bmorris3)

Reply:
I believe so, it looked like this value comes from code that would have been affected by that change.